src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -629,6 +629,16 @@ class CreateTableStatementSegment(BaseSegment):
     )
 
 
+@spark3_dialect.segment()
+class CreateHiveFormatTableStatementSegment(hive_dialect.get_segment("CreateTableStatementSegment")):  # type: ignore
+    """A `CREATE TABLE` statement using Hive format.
+
+    https://spark.apache.org/docs/latest/sql-ref-syntax-ddl-create-table-hiveformat.html
+    """
+
+    type = "create_table_statement"
```

Comment:
Can see this hasn't actually changed but the position has been moved. What's the motivation for this? Just makes the diff more messy.

Reply:
I've done this to maintain the same order of functions listed in the [SparkSQL reference](https://spark.apache.org/docs/latest/sql-ref-syntax.html). I'll separate it into another PR in the future